### PR TITLE
Re-enable rspec tests and speed them up

### PR DESCRIPTION
**Pull request by alto**
See [story ticket](https://vimaly.com/#j8mqm/t/www_site_Fix_the_rspec_tests_to_not_time_out/kvqPEjW6lkUmdumQX).

### CODE REVIEW

File: spec/features/sitemap_spec.rb [hunk 1]
```diff
@@ -28,25 +28,25 @@
     end
   end
 
-  it "should have the right length of links" do
-    locales.each do |locale|
-      visit localize_path('sitemap.xml', locale_id: locale[:id])
+  # Capybara.app.data.locales.each do |locale|
```
Comment (alto) [inline spec/features/sitemap_spec.rb]: Re-enable these tests!

File: spec/features/sitemap_spec.rb [hunk 2]
```diff
@@ -108,27 +108,27 @@
     end
   end
 
-  it "should have all landing pages" do
-    locales.each do |locale|
-      visit localize_path('sitemap.xml', locale_id: locale[:id])
+  # Capybara.app.data.locales.each do |locale|
```
Comment (alto) [inline spec/features/sitemap_spec.rb]: Re-enable these tests!

File: spec/support/helpers/landing_pages.rb [hunk 3]
```diff
@@ -21,6 +21,6 @@ def get_landing_pages(locale_obj = Capybara.app.default_locale_obj)
       model
     end
 
-    return collection
+    collection
```
Comment (kylorhall) [inline spec/support/helpers/landing_pages.rb]: But then how will I know it returns it?  You'll be making thousands of commits like these, good luck.

Comment (alto) [reply spec/support/helpers/landing_pages.rb]: Ruby always returns the last result. And I'm not going through all source files and change those things. Only when I touch the file anyhow.

File: spec/features/blog_posts_page_spec.rb [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 describe 'Blog Post Pages', :type => :feature do
   before :all do
-    @posts = get_blog_posts()
+    @posts = get_blog_posts(order: :youngest_first, limit: 10)
```
Comment (michael-sands) [inline spec/features/blog_posts_page_spec.rb]: 👍

**Pull request merged**